packages/esm-appointments-app/src/constants.ts

```diff
@@ -3,3 +3,51 @@ export const basePath = '/appointments';
 export const spaBasePath = `${window.spaBase}/home`;
 export const omrsDateFormat = 'YYYY-MM-DDTHH:mm:ss.SSSZZ';
 export const appointmentLocationTagName = 'Appointment Location';
+
+export const datePickerPlaceHolder = 'dd/mm/yyyy';
+export const dateFormat = 'DD/MM/YYYY';
+export const datePickerFormat = 'd/m/Y';
+export const weekDays = [
+  {
+    id: 'MONDAY',
```

Comment:
Is there a way we could rework this without the `id`? The label seems just as useful (and English is very unlikely to change the name of "Monday" any time soon).

Reply:
this was stuff I just copied directly from the patient-chart when copying the appointment form over, I can look into simplifying it.